pointercrate-user-api/src/endpoints/auth.rs

```diff
@@ -62,16 +111,6 @@ pub async fn invalidate(mut auth: BasicAuth) -> Result<Status> {
     Ok(Status::NoContent)
 }
 
-#[rocket::get("/verify_email?<token>")]
-pub async fn verify_email(mut auth: TokenAuth, token: &str) -> Result<&'static str> {
-    let email = auth.user.validate_change_email_token(token)?;
```

Comment:
There's a few more of these old left-overs that can be deleted, like `pointercrate_user::auth::generate_change_email_token`, could you also track those down for deletion? :pleading_face:

Reply:
Just did this myself in 23ec44fa40b64f217d77863a443465655b7d2d31